README.md

```diff
@@ -30,9 +31,17 @@ possible to avoid Docker altogether and run everything directly on the host mach
 
 Before starting the installation process, please have a look at the system requirements and the note on security.
```

Comment:
Before starting the installation process, please have a look at the following sections on the distinction between Laika and the Trustlines Blockchain, system requirements, and security.

Reply:
:heavy_check_mark: